### PR TITLE
#193 Theme added(Winter)

### PR DESCRIPTION
**Pull request by anushri20**

### CODE REVIEW

File: home-page/themes.js [hunk 1]
```diff
@@ -189,6 +189,28 @@ let abyss = {
   "section-header-color": '#ba301e',
   "footer-bg": "hsl(0,0%,5%)"
 };
+let winter = {
+  themeName: "winter",
```
Comment (terrifricker) [inline home-page/themes.js:193]: `let winter = {
  themeName: "Winter",`

File: index.html [hunk 2]
```diff
@@ -40,6 +40,8 @@ <h3>Home</h3>
               <a class="dropdown-item" href="#" id="summerdayTheme">Summer Day Theme</a>
               <a class="dropdown-item" href="#" id="pumpkinSpiceTheme">Pumpkin Fall Theme</a>
               <a class="dropdown-item" href="#" id="abyssTheme">Abyss Theme</a>
+              <a class="dropdown-item" href="#" id="winterTheme">winter Theme</a>
```
Comment (terrifricker) [inline index.html:43]: Capitalize Winter in the dropdown menu to keep capitalization consistent with other items in the menu ... `id="winterTheme">Winter Theme</a>`

**Comment by terrifricker** [comment]
Thank you for the new theme.  The colors are great for a Winter theme.

**Review by terrifricker — CHANGES_REQUESTED** [review]
You've made good color choices to represent Winter.  I like that they are so cool-colored.  I would like to request one change:

In the index.html file, please capitalize Winter in the drop menu link.  I've marked the spot in the file here in the pull request.

Thank you!

**Comment by anushri20** [comment]
Thank you for pointing the changes, I just didn't notice it. I will make the change asap.

**Review by terrifricker — APPROVED** [review]
Wonderful!

**Pull request merged**